dealership/customers.py

```diff
@@ -1,11 +1,20 @@
 class Person(object):
+
     def __init__(self, first_name, last_name, email):
-        pass
+        self.first_name = first_name
+        self.last_name = last_name
+        self.email = email
 
 
 class Customer(Person):
-    pass
+
+    def is_employee(self):
+
```

Comment:
I'm having trouble finding any faults with your logic or thinking in this project, so I'll point out something that I find myself struggling to remember as well: https://www.python.org/dev/peps/pep-0008/

Some of your code is running outside of the Pep8 guidelines...worth looking into.